### PR TITLE
Fix Spotless license update

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Description**:

Adress multiple problems with Spotless formatting keeping us from bumping the copyright year. 

* Add go and protobuf files to Spotless formatting
* Fix incorrectly formatted license headers
* Fix lambda warnings in Kotlin
* Fix Spotless exception due to not matching license header pattern in web3 k6
* Fix Spotless not formatting Kotlin and Kotlin script files
* Fix Spotless not formatting files outside sub-modules
* Remove duplicate license headers
* Remove XML Spotless formatting as we only have one such file

**Related issue(s)**:

**Notes for reviewer**:

Will follow up with two PRs: a repo wide formatting fix and a repo wide copyright bump

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by jascks — DISMISSED** [review]
LGTM

**Review by edwin-greene — DISMISSED** [review]
Looks good

File: build.gradle.kts [hunk 1]
```diff
@@ -1,9 +1,6 @@
-/*-
- * ‌
- * Hedera Mirror Node
- * ​
- * Copyright (C) 2019 - 2023 Hedera Hashgraph, LLC
- * ​
+/*
+ * Copyright (C) 2022-2023 Hedera Hashgraph, LLC
```
Comment (jascks) [inline build.gradle.kts:2]: 2024? Not sure on "the chicken or the egg" aspect of this as to when files were committed vs when spotless was able to update the copyright within the various types of files.

Comment (steven-sheehy) [reply build.gradle.kts:2]: See my comment in the description:

> Will follow up with two PRs: a repo wide formatting fix and a repo wide copyright bump

Comment (jascks) [reply build.gradle.kts:2]: Okay, just read notes for reviewer... Never mind. 😄

**Review by xin-hedera — APPROVED** [review]
LGTM

**Review by jascks — APPROVED** [review]
LGTM

**Pull request merged**